### PR TITLE
adding option for self provisioned agents

### PR DESCRIPTION
**Pull request by prmittal**

### CODE REVIEW

File: Tasks/RunLoadTest/task.json [hunk 1]
```diff
@@ -59,7 +59,18 @@
...
-    }
+    },
+    {
+      "name": "MachineType",
+      "type": "radio",
+      "label": "Run load test using",
+      "required": true,
+      "defaultValue": "0",
+      "options": {
+        "2": "Self-provisioned agents",
```
Comment (dpksinghal) [inline Tasks/RunLoadTest/task.json]: Does the order matter?

**Comment by dpksinghal** [comment]
Approved with one comment.

File: Tasks/RunLoadTest/task.json (hunk 1 above)
Comment (prmittal) [inline Tasks/RunLoadTest/task.json]: No.

Comment (prmittal) [inline Tasks/RunLoadTest/task.json]: No.

---
In reply to: [73512668](https://github.com/Microsoft/vsts-tasks/pull/2158#discussion_r73512668) [](ancestors = 73512668)

**Pull request closed without merging**